mp/src/game/shared/momentum/mom_system_gamemode.h

```diff
@@ -6,6 +6,8 @@
 #define CMomentumPlayer C_MomentumPlayer
 #endif
 
+#define VIEW_SCALE ( g_pGameModeSystem->GameModeIs(GAMEMODE_RJ) ? 1.0f : 0.5f )
```

Comment:
Not sure what others think of this, but I'm not a fan of a non-function looking macro (ie. not `VIEW_SCALE()`) calling a function that could potentially have side-effects. (I know this one doesn't, but still).

My preference would be to make this an inline function instead.
You could also make the viewscale property defined in each gamemode itself via a virtual func, but that might be overkill for now.

Reply:
NON_JUMP_VELOCITY is also of this form. I think I'll look into it after sticky jump has been added to develop since I also want to find a way to improve the check g_pGameModeSystem->GameModeIs(GAMEMODE_RJ) || g_pGameModeSystem->GameModeIs(GAMEMODE_SJ).

Reply:
I personally don't see a problem as it's resolving to a static value, but checking the game mode to return the right one.

I do agree with making it a value defined in the game mode, however, as view scale is directly dependent on the current game mode being RJ/SJ or not.